.github/workflows/relock.yml

```diff
@@ -0,0 +1,44 @@
+name: relock
+
+on:
+  workflow_dispatch: null
+  schedule:
+    - cron:  '15 * * * *'
```

Comment:
Isn't this too often? Every 15 mins? 🤔 
I'd say once or twice a week is more than enough.

Reply:
I have a ton of changes coming through to make this a lot smarter. Sorry to bump you early.

Reply:
Ah ok, no probs!

Reply:
OK I pushed to main for testing. See relock_me.py in the autotick bot directory.